### PR TITLE
[@types/apple-mapkit-js] Update types for MapKitInitOptions and MapConstructorOptions

### PR DESCRIPTION
**Pull request by hkbertoson**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:

- [ X] Provide a URL to documentation or source code which provides context for the suggested changes: [MapkitinitOptions](https://developer.apple.com/documentation/mapkitjs/mapkitinitoptions)
[LoadPriority](https://developer.apple.com/documentation/mapkitjs/mapconstructoroptions/4096368-loadpriority)

### CODE REVIEW

File: types/apple-mapkit-js-browser/index.d.ts [hunk 1]
```diff
@@ -576,6 +583,18 @@ declare namespace mapkit {
...
+
+        /**
+         * A value used to control how MapKit JS prioritizes loading map features. This
+         * can be used to optimize the user's initial map loading experience.
+         * 
+         * Valid values include:
+         *  * `mapkit.Map.LoadPriorities.LandCover`: Prioritizes map tiles, land cover, and borders.
+         *  * `mapkit.Map.LoadPriorities.PointsOfInterest`: Prioritizes points of interest (POIs).
+         *  * `mapkit.Map.LoadPriorities.None`: No special prioritization.
+         *
```
Comment (vinayakkulkarni) [inline types/apple-mapkit-js-browser/index.d.ts:595]:
```suggestion
```

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/apple-mapkit-js-browser/apple-mapkit-js-browser-tests.ts
1. types/apple-mapkit-js-browser/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 819fa27e5629d65654068bf1c7a4a2078be35476
</p>

File: types/apple-mapkit-js-browser/apple-mapkit-js-browser-tests.ts [hunk 2]
```diff
@@ -5,10 +5,13 @@ mapkit.init({
     },
 });
 
+
```
Comment (vinayakkulkarni) [inline types/apple-mapkit-js-browser/apple-mapkit-js-browser-tests.ts:8]:
```suggestion
```

File: types/apple-mapkit-js-browser/index.d.ts [hunk 3]
```diff
@@ -59,7 +59,14 @@ declare namespace mapkit {
          * An ID that indicates the preferred language in which to display map
          * labels, controls, directions, and other text.
          */
+
```
Comment (vinayakkulkarni) [inline types/apple-mapkit-js-browser/index.d.ts:62]:
```suggestion
```

File: types/apple-mapkit-js-browser/index.d.ts [hunk 4]
```diff
@@ -59,7 +59,14 @@ declare namespace mapkit {
...
         language?: string | undefined;
+
+
```
Comment (vinayakkulkarni) [inline types/apple-mapkit-js-browser/index.d.ts:65]:
```suggestion
```

**Review by vinayakkulkarni — CHANGES_REQUESTED** [review]

File: types/apple-mapkit-js-browser/index.d.ts [hunk 5]
```diff
@@ -60,6 +60,11 @@ declare namespace mapkit {
          * labels, controls, directions, and other text.
          */
         language?: string | undefined;
+
+        /**
+         * An array of strings specifying the MapKit JS libraries to load at initialization. 
+         */
+        libraries?: string[] | undefined;
```
Comment (MysteryBlokHed) [inline types/apple-mapkit-js-browser/index.d.ts:67]: If this array is never modified, then it should be marked `readonly`:

```suggestion
        libraries?: readonly string[] | undefined;
```

**Review by Anonymous4078 — APPROVED** [review]

**Comment by hkbertoson** [comment]
Ready to Merge

**Pull request merged**